plugins/main/public/controllers/management/components/management/statistics/statistics-dashboard-remoted.js

```diff
@@ -11,80 +11,13 @@
  * Find more information about this on the LICENSE file.
  */
 import React from 'react';
-import WzReduxProvider from '../../../../../redux/wz-redux-provider';
-import KibanaVis from '../../../../../kibana-integrations/kibana-vis';
-import { EuiFlexGroup, EuiFlexItem, EuiPanel } from '@elastic/eui';
 import { useBuildStatisticsVisualizations } from './hooks';
+import { DashboardListenerEngineStatistics } from '../../../../../components/overview/server-management-statistics/dashboards/dashboard_listener_engine';
 
 export function WzStatisticsRemoted({
   clusterNodeSelected,
   refreshVisualizations,
 }) {
   useBuildStatisticsVisualizations(clusterNodeSelected, refreshVisualizations);
-  return (
-    <div>
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiPanel style={{ height: '400px' }}>
-            <EuiFlexGroup>
-              <EuiFlexItem>Total number of bytes received</EuiFlexItem>
-            </EuiFlexGroup>
-            <EuiFlexGroup>
-              <EuiFlexItem>
-                <div style={{ height: '343px' }}>
-                  <WzReduxProvider>
-                    <KibanaVis
-                      visID={'Wazuh-App-Statistics-remoted-Recv-bytes'}
-                      tab={'statistics'}
-                    ></KibanaVis>
-                  </WzReduxProvider>
-                </div>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiPanel>
-        </EuiFlexItem>
-        <EuiFlexItem>
-          <EuiPanel style={{ height: '400px' }}>
-            <EuiFlexGroup>
-              <EuiFlexItem>Events sent to Analysisd</EuiFlexItem>
-            </EuiFlexGroup>
-            <EuiFlexGroup>
-              <EuiFlexItem>
-                <div style={{ height: '343px' }}>
-                  <WzReduxProvider>
-                    <KibanaVis
-                      visID={'Wazuh-App-Statistics-remoted-event-count'}
-                      tab={'statistics'}
-                    ></KibanaVis>
-                  </WzReduxProvider>
-                </div>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiPanel>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiPanel style={{ height: '400px' }}>
-            <EuiFlexGroup>
-              <EuiFlexItem>TCP sessions</EuiFlexItem>
-            </EuiFlexGroup>
-            <EuiFlexGroup>
-              <EuiFlexItem>
-                <div style={{ height: '343px' }}>
-                  <WzReduxProvider>
-                    <KibanaVis
-                      visID={'Wazuh-App-Statistics-remoted-tcp-sessions'}
-                      tab={'statistics'}
-                    ></KibanaVis>
-                  </WzReduxProvider>
-                </div>
-              </EuiFlexItem>
-            </EuiFlexGroup>
-          </EuiPanel>
-        </EuiFlexItem>
-      </EuiFlexGroup>
-    </div>
-  );
+  return <DashboardListenerEngineStatistics />;
```

Comment:
**question**: are some props misisng? https://github.com/wazuh/wazuh-dashboard-plugins/pull/6542/files#diff-6a23b7fe72fa94f3311a1dd53387f3ca523fdf8cd375d4ac00c81565496bcb6fR41-R43

Reply:
`statistics-dashboard-remoted.js` and `statistics-dashboard-analysisd.js` are no longer used. They are removed.